### PR TITLE
[TW-352] Configuring SSO for a team refresh

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Removed deprecated key-values from frontmatter
* Added intro and TOC
* Updated UI instructions and screenshots
* Updated terms and styling to be consistent throughout the page (and SSO section)
* Expanded on sub-sections in the **Managing user accounts** section, adding more details
* Added Troubleshooting section using content from Lessonly
* Ran Vale on the page, and found no major issues

### CODE REVIEW

File: src/pages/docs/administration/sso/admin-sso.md [hunk 1]
```diff
@@ -21,52 +18,74 @@ contextual_links:
...
 1. Select the authentication type.
 1. Enter an authentication name that's identifiable to your team.
-1. Select **Proceed**.
+1. Select **Continue** to [configure the IdP details](#configuring-the-identity-provider-details).
 
-<img src="https://assets.postman.com/postman-docs/add-auth-method.jpg" width="350px" alt="Authentication Method"/>
+    <img src="https://assets.postman.com/postman-docs/v10/add-auth-method-v10.jpg" width="600px" alt="Authentication Method"/>
 
-> Always check with your authentication provider dashboard or your IT support staff for the correct information to complete a modal.
+    > Always check with your authentication provider dashboard or your IT support staff for the correct information to complete a modal.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/admin-sso.md:43]: Both "dialog" and "modal" are discouraged per the [A-Z word list](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3346170709/A-Z+word+list#D). How about: 

Always check with your authentication provider or your IT support staff for the correct information to complete SSO configuration.

File: src/pages/docs/administration/sso/admin-sso.md [hunk 2]
```diff
@@ -21,52 +18,74 @@ contextual_links:
...
 
-In the **Service Provider Details (Postman)** screen, the **Entity ID**, the **Login URL**, and the **ACS URL** are already populated.
+### Configuring the identity provider details
 
-As a next step, you must fill in the details in the **Identity Provider Details** section. Enter your authentication certificate from your identity provider in "X.509 Certificate".
+After [adding the authentication method](#configuring-single-sign-on), you can configure the IdP details.
 
-   [![details](https://assets.postman.com/postman-docs/server-provider-details.jpg)](https://assets.postman.com/postman-docs/server-provider-details.jpg)
+> To continue configuring the IdP details later, select **Configure Later**. When you're ready to continue configuring the IdP details, see [Editing SSO settings](#editing-sso-settings).
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/admin-sso.md:49]:
```suggestion
> To configure the IdP details later, select **Configure Later**. When you're ready to continue configuring the IdP details, see [Editing SSO settings](#editing-sso-settings).
```

**Review by mark-anthony-postman — APPROVED** [review]
Changes look great! 🌟 Just a couple small comments 😀

File: src/pages/docs/administration/sso/admin-sso.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/admin-sso.md]: Good catch! This is now fixed.

**Review by claireriley — APPROVED** [review]
LGTM! Thanks @MatthewPinsker-postman! ⭐  

My only comment is maybe it's worth considering splitting the page? I think "Managing user accounts" could be its own page and break things up a little further, but it reads very well either way

**Review by PriyambadaMadala — APPROVED** [review]
Changes looks good . 
Much appreciate the effort on Troubleshooting section

File: src/pages/docs/administration/sso/admin-sso.md [hunk 3]
```diff
@@ -21,52 +18,74 @@ contextual_links:
 
 > **[SSO is available on Postman Professional and Enterprise plans.](https://www.postman.com/pricing)**
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/sso/admin-sso.md:19]: SSO is available only on the Enterprise plan. 

Only GSuite SSO is available on the Professional plan.

File: src/pages/docs/administration/sso/admin-sso.md [hunk 4]
```diff
@@ -21,52 +18,74 @@ contextual_links:
...
-* [Setting up SSO with Ping Identity](/docs/administration/sso/ping-identity/)*
+* [Setting up SSO with Okta](/docs/administration/sso/okta/)*
 
-* [Setting up SSO with Azure AD](/docs/administration/sso/azure-ad/)*
+* [Setting up SSO with OneLogin](/docs/administration/sso/onelogin/)*
 
-* [Setting up SSO with Microsoft AD FS](/docs/administration/sso/microsoft-adfs/)*
+* [Setting up SSO with Ping Identity](/docs/administration/sso/ping-identity/)*
 
 &#42; Only available on Enterprise plans.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/sso/admin-sso.md:73]: We also have the option to setup custom SSO with the SAML 2.0 option

Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/admin-sso.md:73]: It's my understanding that users configuring SSO with [Azure AD](https://learning.postman.com/docs/administration/sso/azure-ad/) must select **SAML 2.0** as the authentication type.

It's also my understanding that some IdP configurations have their own custom SAML instructions, such as [Okta](https://learning.postman.com/docs/administration/sso/okta/#configuring-sso-using-a-custom-saml-app).

Could you please clarify how users can set up custom SAML by selecting **SAML 2.0** as the authentication type? (Currently, it looks like we only briefly mention this in [Intro to SSO](https://learning.postman.com/docs/administration/sso/intro-sso/#identity-providers-supported), but this mention points to [Configuring SSO for a team](https://learning.postman.com/docs/administration/sso/admin-sso/).)

Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/admin-sso.md:75]: I added a mention about Custom SAML to this section. I linked the text to point to the following section for more details: https://learning.postman.com/docs/administration/sso/intro-sso/#sso-setup-for-saml-20-compliant-idps

It looks like we don't currently have a page/section that explains the process for setting up Custom SAML using the `SAML 2.0` option. I'll bring this up with my team.

Comment (ArjunSingh-PM) [reply src/pages/docs/administration/sso/admin-sso.md:75]: If the identity provider a customer uses is not one of the listed SSO identity providers we expressly support (like Azure AD), the SAML 2.0 option can be used to set up SSO with any identity provider that conforms to the SAML 2.0 specification.

File: src/pages/docs/administration/sso/admin-sso.md (hunk 3 above)
Comment (ArjunSingh-PM) [reply src/pages/docs/administration/sso/admin-sso.md:19]: Thanks for the details. This should be sufficient.

**Review by ArjunSingh-PM — APPROVED** [review]
The changes look good.

File: src/pages/docs/administration/sso/admin-sso.md (hunk 4 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/admin-sso.md:75]: Thanks for the details. I created a Jira ticket for these changes: https://postmanlabs.atlassian.net/browse/TW-956

**Pull request merged**